tests/test_amity.py

```diff
@@ -0,0 +1,215 @@
+        members = office.members
+        self.assertEqual(len(members), 0)
+
+    def test_print_allocations_to_terminal(self):
+        '''test that allocated people are printed to the terminal'''
+        self.amity.create_room('hogwarts', 'office')
+        self.amity.create_room('php', 'livingspace')
+        self.amity.add_person('sam maina', 'staff')
+        self.amity.add_person('tom wilkins', 'fellow')
+        self.amity.add_person('valt honks', 'fellow', 'Y')
+        self.amity.print_allocations(None)
+
+    def test_print_allocations_file(self):
+        '''test that allocated people are printed to a file'''
+        self.amity.create_room('hogwarts', 'office')
+        self.amity.create_room('php', 'livingspace')
+        self.amity.add_person('sam maina', 'staff')
+        self.amity.add_person('tom wilkins', 'fellow')
+        self.amity.add_person('valt honks', 'fellow', 'Y')
+        self.amity.print_allocations('allocations.txt')
+
+    def test_people_without_rooms_are_added_to_unallocated_list(self):
+        '''test that people without rooms are added to unallocated list'''
+        unallocated = self.amity.unallocated
+        self.assertEqual(len(unallocated), 0)
+        self.amity.create_room('hogwarts', 'office')
+        self.amity.create_room('php', 'livingspace')
+        self.amity.add_person('sam gaamwa', 'fellow')
+        self.amity.add_person('tom wilkins', 'fellow')
+        self.amity.add_person('tom shitonde', 'fellow', 'Y')
+        self.amity.print_un_allocated(None)
+        unallocated = self.amity.unallocated
+        self.assertEqual(len(unallocated), 2)
+
+    def test_print_unallocated_people_to_terminal(self):
+        '''test that unallocated people are printed to a the terminal'''
+        self.amity.create_room('hogwarts', 'office')
+        self.amity.create_room('php', 'livingspace')
+        self.amity.add_person('sam maina', 'staff')
+        self.amity.add_person('tom wilkins', 'fellow')
+        self.amity.add_person('valt honks', 'fellow', 'Y')
+        self.amity.print_un_allocated(None)
+
+    def test_print_unallocated_people_to_a_file(self):
+        '''test that unallocated people are printed to a file'''
+        self.amity.create_room('hogwarts', 'office')
+        self.amity.create_room('php', 'livingspace')
+        self.amity.add_person('sam maina', 'staff')
+        self.amity.add_person('tom wilkins', 'fellow')
+        self.amity.add_person('valt honks', 'fellow', 'Y')
+        self.amity.print_un_allocated('unallocated.txt')
+
+    def test_members_of_a_room_are_printed(self):
+        '''test that members of a room are printed'''
+        self.amity.create_room('hogwarts', 'office')
+        self.amity.create_room('php', 'livingspace')
+        self.amity.add_person('sam gaamwa', 'fellow')
+        self.amity.add_person('sam maina', 'staff')
+        self.amity.add_person('tom wilkins', 'fellow')
+        self.amity.print_room('hogwarts')
```

Comment:
you have a lot of tests that don't assert anything. please rectify this and actually test for actual conditions.